### ISSUE TITLE
[BUG]: [Child] Admin | Sync Logs throws error always

### ISSUE THREAD

**Issue by rkarthi37** [microsoft/powerapps-tools#546]
**Describe the bug**
I have followed the deployment document and everything got succeeded as per the article. But when i turned on the flows, i get error Blob key 20201028014116775046578$20201028022417407051751$audit_general$Audit_General$na0043 in the url is invalid on GetContentDetails action. Please help me to fix the issue

COE StarterKit - 
 - Center of Excellence - Audit Logs
 - 1.0.0.20(In 1.0.0.17 also same error)
 - [Child] Admin | Sync Logs

**To Reproduce**
1. Go to Admin | Sync Audit Logs
2. Run it
3. Open any instance of [Child] Admin | Sync Logs

**Expected behavior**
Flow should be completed without error and audit logs would be processed accordingly

**Screenshots**
![audit log error](https://user-images.githubusercontent.com/73622263/97514825-0ec3be80-1966-11eb-8ea8-c03381e2c3bb.png)

**Additional context**

**Comment by JeneferM-MSFT** [comment]
Can you please update parent flow to point to your tenant id? 
![image](https://user-images.githubusercontent.com/62659043/97708254-796c1b80-1a86-11eb-9d29-5391d6ee5433.png)

**Comment by rkarthi37** [comment]
Thanks a lot for discovering it. I just followed the article and was wondering what did i do wrong. I have changed tenant id and flows are working fine now. one more query on the Center of Excellence - Audit Logs. I earlier used audit log by basic authentication. But that had lot of throttling issues, and if there are multiple calls to custom connector(even 3-4 calls at a time) will fail. In Our PROD tenant, sometimes we have 200000 records of launch powerapp logs for a day. I could see there are no more throttling issues with new one. But will the Management API custom connector able to process such huge volume of logs(0.2 Million records/day).?
